hack/generator/pkg/astbuilder/assignments.go

```diff
@@ -42,3 +42,50 @@ func SimpleAssignmentWithErr(lhs dst.Expr, tok token.Token, rhs dst.Expr) *dst.A
 		},
 	}
 }
+
+// SimpleIf creates a simple if statement with a single statement in each branch
+//
+// if <condition> {
+//     <trueBranch>
+// } else {
+//     <falseBranch>
+// }
+//
+func SimpleIf(condition dst.Expr, trueBranch dst.Stmt, falseBranch dst.Stmt) *dst.IfStmt {
+	result := &dst.IfStmt{
+		Cond: condition,
+		Body: StatementBlock(trueBranch),
+		Else: StatementBlock(falseBranch),
```

Comment:
These seem redundant since you're overwriting `Body` and `Else` below.
```suggestion
		Body: StatementBlock(trueBranch),
		Else: StatementBlock(falseBranch),
```